Update serve mode to accept --tags and --metadata flags

flimzy opened the pull request:
### Description:
Pass `--tags` and `--metadata` flag values from `fyne serve` to package phase.

Fixes #5074

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

andydotxyz reviewed (changes requested):
Hmm, I think some of those `p`s should be `s`.
Double check that you tested the right thing, not sure how something like that slipped in.

flimzy commented:
:facepalm: 

Muscle memory had me type `git push origin -f serveMetadata` when I fixed that, which of course failed, because I don't have write permission to this repo. And I didn't notice the error message. I've now updated the _correct_ remote.

andydotxyz reviewed (approved):
Great thanks
